modules/dxp/apps/portal-search-tuning/portal-search-tuning-gsearch-configuration-web/src/main/resources/META-INF/resources/js/components/Fragment.js

```diff
@@ -161,16 +146,14 @@ export default function Fragment({
 			</ClayList>
 
 			{expand && (
-				<div className="configuration-editor">
-					<AceEditor
-						onChange={handleChange}
-						onRender={({editorElement, editorTextInput}) => {
-							editorElementRef.current = editorElement;
-							editorTextInputRef.current = editorTextInput;
-						}}
-						value={value}
-					/>
-				</div>
+				<AceEditor
+					onBlur={(value) => setValue(value)}
+					onRender={({editorElement, editorTextInput}) => {
+						editorElementRef.current = editorElement;
+						editorTextInputRef.current = editorTextInput;
+					}}
+					value={value}
```

Comment:
What if we got rid of the whole `value` state and this was `value={valueRef.current}` instead? (Meaning adding onChange back again)

Reply:
Oh ok I see what you mean now, my bad, 😭 looks like it works with just `valueRef.current`. I'll add it in!